packages/block-library/src/site-title/block.json

```diff
@@ -20,6 +20,7 @@
 		"fontSize": true,
 		"lineHeight": true,
 		"__experimentalFontFamily": true,
+		"__experimentalSelector": ".wp-block-site-title, .wp-block-site-title > a",
```

Comment:
Why do we target the "inner a" here?

Reply:
The goal is to change the colour of the site title, which is an anchor tag wrapped in a heading.

By default, an anchor tag does not inherit its parent's colour settings if an href attribute is present. So to change the appearance of the title we target the inner anchor.

Or well, that's what was done here. :)

Reply:
> By default, an anchor tag does not inherit its parent's colour settings if an href attribute is present. So to change the appearance of the title we target the inner anchor.

It does break font size and transform styles for the block though. I'm reverting that here #30558 to solve these. Maybe we can just add `color: inherit` for the link in the style of the block?

Reply:
In my tests though, the color is applied properly even with the revert because it actually applies  the link color (which is the right thing to do there)

Reply:
I also tested the `color`  in the revert and worked well.